src/main/java/frc/robot/commands/drivetrain/DrivetrainDefaultCommand.java

```diff
@@ -117,12 +119,26 @@ else if (Robot.overallState == OverallState.SCORING) {
                 )
             );
         }
-
     }
 
-    private double getRValueSquareWithField() {
-        // TODO: retrieve the R value for the drivetrain that allows the robot to square with the field
-        return 0;
+    private double getAngularVelocityForScoringAlign() {
+        // Assumes that the robot's initial rotation (0) is aligned with the scoring nodes
+        // double currentRotationOffset = MathUtil.angleModulus(Robot.DRIVE_TRAIN_SUBSYSTEM.getOdometryRotation().getRadians());
+        // double pidCalculation = _scoringRotationAlignPID.calculate(Math.abs(currentRotationOffset));
+        // double angularVelocity = pidCalculation * (currentRotationOffset < 0 ? -1 : 1);
+        // if (angularVelocity > DrivetrainSubsystem.MAX_ANGULAR_VELOCITY_RADIANS_PER_SECOND) {
+        //     return DrivetrainSubsystem.MAX_ANGULAR_VELOCITY_RADIANS_PER_SECOND;
+        // }
+        // This will need to be tested
+        double angularVelocity = _scoringRotationAlignPID.calculate(Robot.DRIVE_TRAIN_SUBSYSTEM.getOdometryRotation().getRadians()) * DrivetrainSubsystem.MAX_ANGULAR_VELOCITY_RADIANS_PER_SECOND;
+        boolean isAngularVelocityNegative = angularVelocity < 0 ? true : false;
```

Comment:
I think this is a good example of where ternary operators work well :)

However if we change `isAngularVelocityNegative` to `double velocityDirection` and make it `-1` or `1` then you can simplify a some of the future statements. e.g.
`return isAngularVelocityNegative ? DrivetrainSubsystem.MAX_ANGULAR_VELOCITY_RADIANS_PER_SECOND * -1 : DrivetrainSubsystem.MAX_ANGULAR_VELOCITY_RADIANS_PER_SECOND;`
becomes
`return DrivetrainSubsystem.MAX_ANGULAR_VELOCITY_RADIANS_PER_SECOND * velocityDirection;`